docs/docs/api/matrix/api-reference.md

```diff
@@ -73,24 +73,34 @@ However, there are important limitations of the `/sources_to_targets` service's
 
 ## Outputs of the matrix service
 
-If a matrix request has been named using the optional `&id=` input, then the name will be returned as a string `id`.
-
-These are the results of a request to the Time-Distance Matrix service.
+Depending on the `verbose` (default: `true`) request parameter, the result of the Time-Distance Matrix service is different.
+In both (`"verbose": true` and `"verbose": false`) cases, these parameters are present:
 
 | Item | Description |
 | :---- | :----------- |
-| `sources_to_targets` | Returns an array of time and distance between the sources and the targets. The array is **row-ordered**. This means that the time and distance from the first location to all others forms the first row of the array, followed by the time and distance from the second source location to all target locations, etc. |
-| `distance` | The computed distance between each set of points. Distance will always be 0.00 for the first element of the time-distance array for `one_to_many`, the last element in a `many_to_one`, and the first and last elements of a `many_to_many`. |
-| `time` | The computed time between each set of points. Time will always be 0 for the first element of the time-distance array for `one_to_many`, the last element in a `many_to_one`, and the first and last elements of a `many_to_many`.  |
-| `to_index` | The destination index into the locations array. |
-| `from_index` | The origin index into the locations array. |
-| `date_time`  | (optional) If the date_time was valid for an origin, `date_time` will return the local time at the destination. |
-| `locations` | The specified array of lat/lngs from the input request.
-| `units` | Distance units for output. Allowable unit types are mi (miles) and km (kilometers). If no unit type is specified, the units default to kilometers. |
-| `warnings` (optional) | This array may contain warning objects informing about deprecated request parameters, clamped values etc. | 
+| `id`                 | Name of the request. Included only if a matrix request has been named using the optional `id` input. |
+| `algorithm`          | The algorithm used to compute the results. Can be `"timedistancematrix"`, `"costmatrix"` or `"timedistancbssematrix"` |
```

Comment:
Looks like there is a typo, but it is also present in the actual response sent.
https://github.com/valhalla/valhalla/blob/e5e695f5209440aabea02004fa3999f49ca154de/src/proto_conversions.cc#L13
```suggestion
| `algorithm`          | The algorithm used to compute the results. Can be `"timedistancematrix"`, `"costmatrix"` or `"timedistancebssmatrix"` |
```

Reply:
Yes, that is where I got this value from.
Since I am only changing the docs, I did not correct the typo in the app.